### PR TITLE
Added github action to update release-notes based on an event

### PR DESCRIPTION
**Pull request by shubhang93**

### CODE REVIEW

File: .github/workflows/ziggurat-release-event.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    types: [ziggurat-release-event]
+
+jobs:
+  changelog-copy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Ziggurat repo
+        uses: actions/checkout@v2
+        with:
+          repository: shubhang93/ziggurat #TODO change it to gojek/ziggurat
```
Comment (hogaur) [inline .github/workflows/ziggurat-release-event.yml]: lets change to gojek/ziggurat, what's stopping us?

File: .github/workflows/ziggurat-release-event.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          token: ${{secrets.GH_TOKEN}}
+      - name: Copy changelog
+        run: |
+          cp ./CHANGELOG.md /tmp/CHANGELOG.md
+
+      - name: Checkout Ziggurat web repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{secrets.GH_TOKEN}}
```
Comment (mjayprateek) [inline .github/workflows/ziggurat-release-event.yml:23]: shouldn't this be a `GITHUB_TOKEN` as explained here: https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Same comment for #L15

And, probably we don't need github tokens to check out the code at lines 15 and 23. We might need it to make a commit at line 29. 

WDYT? Have we already verified this?

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -1,9 +1,10 @@
-name: Deploy
+name: deploy
 
 on:
   push:
-    branches:
-      [master, google_analytics]
+    branches: [master, google_analytics]
+  workflow_dispatch:
```
Comment (mjayprateek) [inline .github/workflows/deploy.yml:6]: why do we need a workflow dispatch event here?

File: .github/workflows/ziggurat-release-event.yml (hunk 1 above)
Comment (shubhang93) [reply .github/workflows/ziggurat-release-event.yml]: this was just for testing it,  once we obtain the credentials then we can change it, or else the pipelines will keep failing

File: .github/workflows/ziggurat-release-event.yml (hunk 2 above)
Comment (shubhang93) [reply .github/workflows/ziggurat-release-event.yml:23]: So the `GITHUB_TOKEN` can be used to checkout the `ziggurat-web` repo but not the `ziggurat` repo. We wanted to keep a uniform token across all the workflows

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (shubhang93) [reply .github/workflows/deploy.yml:6]: Oh! this was already setup before the handover. Need to check once

**Review by anmol1vw13 — APPROVED** [review]

**Pull request merged**